.github/workflows/check-translations-and-deserialize.yml

```diff
@@ -69,6 +70,87 @@ jobs:
           branch-suffix: short-commit-hash
           base: develop
           delete-branch: true
+          team-reviewers: developer-enablement
+
+      - name: Checkout repository
+        if: steps.fetch-deserialize.outputs.batchesToDeserialize != 0
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
```

Comment:
I'm a little confused, this looks like we're re-checking-out the repository and setting up node again. Was this intended to be in it's own job? Or is this intentional.

Reply:
I'm not sure what this is for, but it looks like this was already here. We might want to consider removing it.